src/main/resources/db/changes/0010-add-available-clolumn.yaml

```diff
@@ -0,0 +1,13 @@
+databaseChangeLog:
+  - changeSet:
+      id: 0010-add-available-column
+      author: chellen
+      comment: add available column to books table
+      changes:
+        - addColumn:
+            tableName: books
+            columns:
+              - column:
+                  name: available
+                  type: boolean
+                  defaultValueBoolean: true
```

Comment:
The functionality breaks after I add this in. 

Liquibase error: liquibase.exception.ValidationFailedException: Validation Failed:
     1 changesets check sum
          db/changes/0010-add-available-clolumn.yaml::0010-add-available-column::chellen was: 8:af4f20bda09b04efe5c08767570fa675 but is now: 8:527b02a7c136234b894e2da91f031da5

Reply:
This will happen if you change a changeset. You can resolve this by either deleting your local DB or by carefully undoing its changes (drop the column added in the changeset, and the log of it in `databasechangelog`)